src/xml2js.coffee

```diff
@@ -92,6 +106,7 @@ class exports.Builder
   buildObject: (rootObj) ->
     attrkey = @options.attrkey
     charkey = @options.charkey
+    cdata = @options.cdata
```

Comment:
I've made that change, and removed the local version of `cdata = @options.cdata`. However, it is not inconsistent with other code where the local variable style has been used (for example, see the use of `attrKey` [here](https://github.com/Leonidas-from-XIV/node-xml2js/blob/3fc65f6abcd185092773e90b442a2fb67e188882/src/xml2js.coffee#L93))

I'll leave it up to you if you'd like to change that style throughout the code base ;)